server/app/services/program/ProgramServiceImpl.java

```diff
@@ -351,6 +359,27 @@ public ErrorAnd<ProgramBlockAdditionResult, CiviFormError> addRepeatedBlockToPro
     return addBlockToProgram(programId, Optional.of(enumeratorBlockId));
   }
 
+  private static ErrorAnd<BlockDefinition, CiviFormError> createEmptyBlockDefinition(
+      long blockId, Optional<Long> maybeEnumeratorBlockId) {
+    String blockName;
+    if (maybeEnumeratorBlockId.isPresent()) {
+      blockName =
+          String.format("Screen %d (repeated from %d)", blockId, maybeEnumeratorBlockId.get());
+    } else {
+      blockName = String.format("Screen %d", blockId);
+    }
```

Comment:
suggestion (non-blocking): 
```suggestion
    String blockName = maybeEnumeratorBlockId.isPresent()
        ? String.format("Screen %d (repeated from %d)", blockId, maybeEnumeratorBlockId.get())
        : String.format("Screen %d", blockId);
```

Reply:
Done!